City-of-Helsinki/berth-reservations-admin
# Change berth number to string

## Description :sparkles:
Change berth number to string.

## Issues :bug:

### Related :handshake:
- https://github.com/City-of-Helsinki/berth-reservations/pull/287
- [VEN-630](https://helsinkisolutionoffice.atlassian.net/browse/VEN-630)
## Testing :alembic:

### Manual testing :construction_worker_man:
- Go to Venepaikat
- Open a harbor
- You should be able to add a new berth with number "1A"
